vulnerabilities/importer.py

```diff
@@ -533,6 +661,7 @@ class AdvisoryDataV2:
     summary: Optional[str] = ""
     affected_packages: List[AffectedPackage] = dataclasses.field(default_factory=list)
     references: List[ReferenceV2] = dataclasses.field(default_factory=list)
+    patches: List[PatchData] = dataclasses.field(default_factory=list)
```

Comment:
@TG1999 We're not using `AdvisoryDataV2` anywhere, IMO we should get rid of dead code if it's no longer needed.

Reply:
@keshav-space agreed!

Reply:
@TG1999 @keshav-space You mean we should get rid of `AdvisoryData`, not `AdvisoryDataV2`?

AdvisoryData is the old version, am I right?

Reply:
@ziadhany  We are not using `AdvisoryDataV2` anywhere, for both v1 and v2 we're using `AdvisoryData`.